pipeline-scripts/buildlib.groovy

```diff
@@ -812,6 +812,24 @@ def parse_record_log( working_dir ) {
     return result
 }
 
+// Find the supported arches for this release
+//
+// @param branch <String>: The name of the branch to get configs
+//   for. For example: 'openshift-4.3'
+//
+// @return arches <List>: A list of the arches built for this branch
+def branch_arches(String branch) {
+    echo("Fetching group config for '${branch}'")
+    def groupConfigSource = "https://gitlab.cee.redhat.com/openshift-art/ocp-build-data/raw/${branch}/group.yml"
```

Comment:
This needs to change back to github, doesn't it?

Reply:
yep https://github.com/openshift/aos-cd-jobs/pull/1997

Reply:
Merged that pr. and fixed this